src/utils/util.js

```diff
@@ -0,0 +1,11 @@
+import { TodoItem } from "../component/todoItem.js";
+
+export function generateTodos(todos) {
+	if (todos === null) {
+		return [];
+	}
+	else {
+		// map 은 새로운 배열을 반환함.
+		return todos.map(todo => new TodoItem(todo.id, todo.text, todo.status));
+	}
```

Comment:
`if` 조건에 빠지면 바로 `return`되니 `else` 문을 굳이 사용할 필요는 없어보입니다. 👍 
```suggestion

    return todos.map(todo => new TodoItem(todo.id, todo.text, todo.status));

```